oraichain/chain.json

```diff
@@ -34,47 +34,67 @@
   },
   "codebase": {
     "git_repo": "https://github.com/oraichain/orai",
-    "recommended_version": "v0.41.7",
+    "recommended_version": "v0.42.4",
     "compatible_versions": [
-      "v0.41.3",
-      "v0.41.4",
-      "v0.41.6",
-      "v0.41.7"
+      "v0.42.4"
     ],
-    "cosmwasm_version": "0.30.2",
+    "cosmos_sdk_version": "v0.45.16",
```

Comment:
Because cosmos sdk version is defined using the legacy property, we'll need to also define it using the replacement property.
```
"sdk": {
  "type": "cosmos",
  "version": "v0.45.16"
}
```

Reply:
Thanks, I've just updated it. Hope it will pass the CI process